src/lib/markbind/src/parsers/componentParser.js

```diff
@@ -181,6 +181,29 @@ function _parseBoxAttributes(element) {
   _parseAttributeWithoutOverride(element, 'icon', true, '_icon');
 }
 
+/*
+ * Dropdowns
+ */
+
+function _parseDropdownAttributes(element) {
+  const el = element;
+  const slotChildren = el.children && el.children.filter(child => _.has(child.attribs, 'slot'));
+  const hasHeaderSlot = slotChildren && slotChildren.some(child => child.attribs.slot === 'header');
+
+  // If header slot is present, the header attribute has no effect, and we can simply remove it.
+  if (hasHeaderSlot) {
+    delete el.attribs.header;
+    return;
+  }
+
+  // header attribute takes priority over text attribute
+  if (_.has(element.attribs, 'header')) {
+    _parseAttributeWithoutOverride(element, 'header', true, '_header');
```

Comment:
`_parseAttributeWithoutOverride` will not override the previous existing `_header` slot, and will delete the attribute.

Remember to delete `el.attribs.text` here as well! Or you could just do away with the `if .. else`